Toolset/home.livecodescript

```diff
@@ -1,4 +1,4 @@
-script "home"
+﻿script "home"
```

Comment:
Can you check the BOM hasn't somehow gone astray? I'm never sure what's gone on when a change looks like this.